tasks/install_shell.sh

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
+
+set -e
```

Comment:
this should be tested a bit, I don't know if this works out of the box on all operating systems

Reply:
`set` is a bash builtin, so assuming the script is executed with Bash (as the shebang requires), then it should work fine.
ref: https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html

Reply:
That wasn't my point. I don't know if the whole code is compatible with `-e`.

Reply:
Either the code does proper error checking, either the test suite needs some fixes :shrug:

From my point of view, the best test case is to release it to the public and if this cause trouble, fix the trouble.